aws-services/reference/package-values.hbs.md

```diff
@@ -28,3 +28,20 @@ The following table lists configuration that applies to the `postgresql` service
+| mysql.infrastructure.security_groups             | _n/a_                 | array   | Security groups for your MySQL databases to belong to                                                                                                                                                                                                                             |
+| mysql.infrastructure.subnet_group.name           | `""`                  | string  | Subnet group for your MySQL databases                                                                                                                                                                                                                                             |
+| mysql.instance_configuration.engine_version      | `"8.0"`               | string  | The MySQL version. For more information, see the [AWS documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_MySQL.html#MySQL.Concepts.VersionMgmt).                                                                                         |
+| mysql.instance_configuration.instance_class      | `db.t3.micro`         | string  | The instance type of the RDS instance. For more information, see the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html).                                                                                                        |
+| mysql.instance_configuration.maintenance_window  | `Mon:00:00-Mon:03:00` | string  | The window to perform maintenance in. Syntax: `ddd:hh24:mi-ddd:hh24:mi`, for example, `Mon:00:00-Mon:03:00`. For more information, see the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.Maintenance.html#Concepts.DBMaintenance). |
+| mysql.instance_configuration.publicly_accessible | `false`               | boolean | Controls whether your instances are publicly accessible                                                                                                                                                                                                                                |
+| mysql.instance_configuration.skip_final_snapshot | `false`               | boolean | Determines whether a final snapshot is created before the instance is deleted. If you specify true, no snapshot is created. If you specify false, a snapshot called final-snapshot-INSTANCE-NAME is created before the instance is deleted.                                            |
+| mysql.provider_config_ref.name                   | `default`             | string  | Name of your ProviderConfig for the `mysql` service                                                                                                                                                                                                                               |
```

Comment:
Do we have both postgres and mysql provider config name as `default`? Doesn't that run into conflict?

Reply:
Given this is provided by the user, there's no issue - they'll just both use the same `ProviderConfig`. I think `default` gets used anyway as it's also Crossplane's default, but it's the best value we can use for our own default too 🤷